taxonomy-manager-rest-server/src/main/java/com/digirati/taxman/rest/server/ServerConceptSchemeResource.java

```diff
@@ -42,4 +42,11 @@ public Response updateConceptScheme(ConceptSchemePath params, @Valid ConceptSche
 
         return Response.noContent().build();
     }
+
+    @Override
+    public Response deleteConceptScheme(ConceptSchemePath params) {
+        conceptSchemes.delete(params.getUuid());
+
+        return Response.noContent().build();
```

Comment:
Should be 410 Gone.

Reply:
https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html

Should it though?

Reply:
Ah, sorry, you're right. 410 Gone is what we should send if someone requested a deleted record.

Reply:
Which, TBH, we don't.

Reply:
We're not even returning 404's right now, so let's leave it out of this PR.